### ISSUE TITLE
https://jesuswordsonly.github.io/index.php contract box

### ISSUE THREAD

**Issue by embed-net** [jesuswordsonly/jesuswordsonly.github.io#3]
Does the contract box work?

I get
```
405 Not Allowed
```

**Comment by jesuswordsonly** [comment]
no. it used to at its original site, but not on this github pages preservation

**Comment by embed-net** [comment]
I'd remove or disable it on the website then - millions of emails have probably gone into the void. I'd also do something on the homepage to say this is your resurrection of the site and not a site of Dougs just so people like me understand.

Thanks for all of your hard work and maybe I can help a bit,
